Tetris/main.qml

```diff
@@ -0,0 +1,78 @@
+import QtQuick 2.12
+import QtQuick.Window 2.12
+import Qt.labs.settings 1.0
+
+Window {
+    id: mainWin
+    visible: true
+    width: 700
+    height: 550
+    title: qsTr("Tetris")
+    maximumHeight: height
+    minimumHeight: height
+    maximumWidth: width
+    minimumWidth: width
+    Settings {
+           property alias x: mainWin.x
+           property alias y: mainWin.y
+       }
+
+    Init{}
+    property var yPos: 25
+    Item{
+        id: element
+        x: 350
+        y: yPos
+        Rectangle{
+            id: one
+            width: 25
+            height: 25
+            color:"red"
+            border.width: 1
```

Comment:
width: 25
height: 25
color:"red"
border.width: 1

looks like common properties, you can create a basic item and inherit those properties